data.json

```diff
@@ -38,6 +38,27 @@
       }
     }
   },
+  "ByteDance/Tiktok": {
+    "url": "https://job.toutiao.com/s/J7LBfB5",
+    "jobs": {
+      "softwareEngineer": {
+        "intern (referral code: VC7FSRM)": {
+          "available": true,
+          "url": "https://careers.tiktok.com/position?project=6854152751179303182&type=3&current=1&limit=100"
+        },
+        "newGrad (referral code: VC7FSRM)": {
+          "available": true,
+          "url": "https://careers.tiktok.com/position?project=6854152751179303182&type=3&current=1&limit=100"
```

Comment:
These are intern positions instead of new grad positions. Do you have a different link for new grad positions?